### DISCUSSION TITLE
Any react-native version(after 0.68) forces to enable new architecture as mandatory?

### DISCUSSION THREAD

**Discussion by Yandamuri** [reactwg/react-native-new-architecture#153] [Q&A] [answered]
My project's current `react-native` version is `0.68.2` but i didn't enable new architecture. I am using many third party libraries in my `react-native` project and many of them do not have support for **new architecture**.

1. How long can I continue to use latest` react-native` versions with out enabling **new architecture**?

2. After enabling the **new architecture**, Should not we use the third party libraries which do not have support for **new architecture**?

**Comment by iBotPeaches** [comment]
1. Tough to tell, but seeing how JSC is still possible alongside Hermes and thats been like 10+ versions - I'd say since New Arch is still marked Experimental - you have quite a while. The adoption will get easier as the interop layer helps bridge the gap between compatible and not compatible libraries.

2. This is the Interop library at work - https://github.com/reactwg/react-native-new-architecture/discussions/135

Your first step though is upgrade a few more RN versions. Personally wouldn't recommend worrying anything about new arch at all until you are on RN72/73.

**Comment by cortinico** [comment, marked as answer]
As @iBotPeaches mentioned, the New Architecture is currently Experimental. As of today, you're free to use any React Native versions without the New Architecture.

We're doing investments to make it easier to adopt the New Architecture with the various Interop Layers (https://github.com/reactwg/react-native-new-architecture/discussions/135). We'll be sharing more on this for React Native 0.73 as well.

In the future, the New Architecture will become the default, once we declare it fully stable and generally available.

**Reply by Yandamuri** [reply]
Thank you so much @cortinico.

**Comment by Yandamuri** [comment]
@iBotPeaches Thank you for your insights. Is it appropriate  to use `herms engine` without enabling new architecture?

**Reply by iBotPeaches** [reply]
Yes - we've been doing it for years.
